common/admin/smb/cluster_test.go

```diff
@@ -144,3 +144,44 @@ func TestClusterSimplePlacement(t *testing.T) {
 		assert.EqualValues(t, c.Placement["label"], "ilovesmb")
 	})
 }
+
+func TestClusterWithRemoteControl(t *testing.T) {
```

Comment:
```suggestion
func TestMarshalUnmarshalWithRemoteControl(t *testing.T) {
```
We already have `TestClusterWithRemoteControl` in _admin_test.go_.